sqs3.tf

```
@@ -0,0 +1,18 @@
+locals {
+  sqs3_name = length(var.sqs3_name_override) > 0 ? var.sqs3_name_override : join("-", [var.name, "1"])
```

Shouldn't it be like so (`3` at the very end)?
```suggestion
  sqs3_name = length(var.sqs3_name_override) > 0 ? var.sqs3_name_override : join("-", [var.name, "3"])
```